### PR TITLE
switch to maven

### PR DESCRIPTION
**Pull request by mariuszs**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,5 @@
-### Gradle ###
-.gradle/
-build/
-classes/
+### Maven ###
+target
```
Comment (szpak) [inline .gitignore]: Missing tailing "/".

File: .mvn/wrapper/maven-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.5.2/apache-maven-3.5.2-bin.zip
```
Comment (szpak) [inline .mvn/wrapper/maven-wrapper.properties]: sha256 distribution checksum check?

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+	<version>0.6.0-SNAPSHOT</version>
+
+	<name>jFairy</name>
+	<description>Java fake data generator</description>
+	<url>https://github.com/Devskiller/jfairy</url>
+
+	<licenses>
+		<license>
+			<name>Apache 2</name>
+			<url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
```
Comment (szpak) [inline pom.xml]: http -> https

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+		<developerConnection>scm:git:git@github.com:Devskiller/jfairy</developerConnection>
+		<url>https://github.com/Devskiller/jfairy</url>
+		<tag>HEAD</tag>
+	</scm>
+
+	<distributionManagement>
+		<snapshotRepository>
+			<id>oss</id>
+			<name>Sonatype Nexus Snapshots</name>
+			<url>http://oss.sonatype.org/content/repositories/snapshots</url>
```
Comment (szpak) [inline pom.xml]: C'mon, I thought Maven supports HTTPS?

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+		</snapshotRepository>
+		<repository>
+			<id>oss</id>
+			<name>Nexus Release Repository</name>
+			<url>http://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
```
Comment (szpak) [inline pom.xml]: The same

File: pom.xml [hunk 6]
```diff
@@ -0,0 +1,301 @@
...
+						</testSource>
+					</testSources>
+				</configuration>
+			</plugin>
+			<plugin>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<configuration>
+					<includes>
+						<include>**/*Spec.java</include>
+						<include>**/*Test.java</include>
```
Comment (szpak) [inline pom.xml]: So many XML to just add Groovy compilation...

**Comment by szpak** [comment]
Multiple tests are failing. Have you built it locally @mariuszs ?

**Comment by mariuszs** [comment]
@szpak Thanks for your help. I hope we all can forgot about gradle soon ;)

**Pull request merged**